src/testing_system/cmd/test_API_pipeline.py

```
@@ -0,0 +1,85 @@
+
+def API_test():
+
+    documents = [
+        RetrievedDocument(
+            id = 1,
+            content="Рейс SU-1231 вылетает 15.05.26 в 09:40 из Санкт-Петербурга в Сочи, время в пути составит 3 часа 20 минут, приземление по местному времени (разница +0).",
+            metadata=None
+        ),
+        RetrievedDocument(
+            id = 2,
+            content="USER-1, извини, я задержусь на презентацию сегодня вечером (начало в 19:00) — поеду не на такси, а на метро из-за перекрытия Тверской улицы, буду около 19:50.",
```

Ruff (0.15.12)

[warning] String contains ambiguous `а` (CYRILLIC SMALL LETTER A). Did you mean `a` (LATIN SMALL LETTER A)?

(RUF001)